src/eval/cast.rs

```diff
@@ -181,6 +182,30 @@ cast_to_value! {
     v: String => Value::Str(v.into())
 }
 
+cast_from_value! {
+    MathClass,
+    "operator" => MathClass::Large,
+    "opening" => MathClass::Opening,
+    "closing" => MathClass::Closing,
+    "binary" => MathClass::Binary,
+    "relation" => MathClass::Relation,
+    "normal" => MathClass::Normal,
+    "punctuation" => MathClass::Punctuation,
```

Comment:
I think we should at least add fence, large, unary, and vary. Also, could you order this and the match below in the same order as [here](https://docs.rs/unicode-math-class/0.1.0/src/unicode_math_class/lib.rs.html#42-58).